yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerSSHLauncher.java

```diff
@@ -74,7 +74,7 @@ public void appendContainerConfig(DockerSlaveTemplate dockerSlaveTemplate, Creat
             createCmd.withCmd("bash", "-c", "/usr/sbin/sshd -D -p " + sshPort);
         }
 
-        createCmd.getPortBindings().add(PortBinding.parse("0.0.0.0::" + sshPort));
+        createCmd.getPortBindings().add(PortBinding.parse("" + sshPort));
```

Comment:
Well, now I see :) - it is required to explicitly create String or compilation fails with "incompatible types: int cannot be converted to java.lang.String"